services/web/server/src/simcore_service_webserver/projects/_projects_service.py

```diff
@@ -221,7 +231,17 @@ async def conditionally_unsubscribe_from_project_logs(
         # the last socket is closed, though another replica may still maintain an active
         # subscription even if no users are connected to it.
         if actually_used_sockets_on_project == 0:
-            await project_logs.unsubscribe(app, project_id)
+            rabbitmq_client = get_rabbitmq_client(app)
```

Comment:
so this is the key change: instead of unsuscribe, you publish an event in the internal webserver